src/rollback_to_stable/rts_history.c

```diff
@@ -58,10 +61,10 @@ __wt_rts_history_delete_hs(WT_SESSION_IMPL *session, WT_ITEM *key, wt_timestamp_
 
         if (!dryrun) {
             __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
-              WT_RTS_VERB_TAG_HS_UPDATE_REMOVE
-              "deleting history store update with stop_timestamp=%s greater than "
-              "stable_timestamp=%s, time_window=%s",
-              __wt_timestamp_to_string(hs_tw->stop_ts, ts_string[0]),
+              WT_RTS_VERB_TAG_HS_UPDATE_REMOVE "deleting history store update for btree_id=%" PRIu32
+                                               "with update stop_timestamp=%s greater than "
+                                               "stable_timestamp=%s, time_window=%s",
+              btree_id, __wt_timestamp_to_string(hs_tw->stop_ts, ts_string[0]),
```

Comment:
These two locations are great. Neither of them are the one I saw (unless it has changed text, which is possible) in the original description of the ticket. Can it be added for that one too or is there a problem doing so like it is at the wrong level?

Reply:
This code has changed quite a bit since the ticket was created - `__rollback_to_stable_btree_hs_truncate` still exists in the form of `__wt_rts_history_btree_hs_truncate`, but it no longer logs anything for each entry. Instead it logs that it truncated records for a specific btree ID once all the work is done. But the version of this function for timestamped tables was still missing that information, so that's what I added here.